test/basic010/run

```diff
@@ -1,10 +1,4 @@
 #!/usr/bin/env bash
-
```

Comment:
Why change this script? And likewise for reg039.

Reply:
Third bullet point. I don't know yet why, it fails on Nixpkgs/Darwin when invoked via the TIMEOUT script.

Reply:
The point of that script is to ensure that there's not an infinite loop. Getting rid of it breaks these tests, so that shouldn't be done. Better to figure out why Nix isn't working with these.